tests/admin_views/tests.py

```diff
@@ -6175,6 +6175,59 @@ def test_selectbox_height_not_collapsible_fieldset(self):
         )
         self.take_screenshot("selectbox-non-collapsible")
 
+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark", "high_contrast"])
+    def test_selectbox_selected_rows(self):
+        from selenium.webdriver import ActionChains
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.common.keys import Keys
+
+        self.admin_login(
+            username="super", password="secret", login_url=reverse("admin:index")
+        )
+        # Create a new user to ensure that no extra permissions have been set.
+        user = User.objects.create_user(username="new", password="newuser")
+        url = self.live_server_url + reverse("admin:auth_user_change", args=[user.id])
+        self.selenium.get(url)
+
+        # Scroll to the User permissions section.
+        user_permissions = self.selenium.find_element(
+            By.CSS_SELECTOR, "#id_user_permissions_from"
+        )
+        ActionChains(self.selenium).move_to_element(user_permissions).perform()
+        self.take_screenshot("selectbox-available-perms-none-selected")
+
+        # Select multiple permissions from the "Available" list.
+        ct = ContentType.objects.get_for_model(Permission)
+        perms = list(Permission.objects.filter(content_type=ct))
+        for perm in perms:
+            elem = self.selenium.find_element(
+                By.CSS_SELECTOR, f"#id_user_permissions_from option[value='{perm.id}']"
+            )
+            ActionChains(self.selenium).key_down(Keys.CONTROL).click(elem).key_up(
+                Keys.CONTROL
+            ).perform()
+
+        ActionChains(self.selenium).send_keys(Keys.TAB).perform()  # Move focus
```

Comment:
![test_selectbox_selected_rows--selectbox-chosen-perms-none-selected--desktop_size](https://github.com/user-attachments/assets/2158a39f-4bc3-4327-8fd6-dd4be47eac5e)

Can we tweak this to click on or focus on something else (maybe something explicit, like the box)? The button has an animation and so it looks like we have an issue with the icon (but it just catches it while it's moving)

Reply:
Sure, I just push a new revision where the focus is moved to the filter input. Screenshots:
![test_selectbox_selected_rows--selectbox-available-perms-some-selected--dark](https://github.com/user-attachments/assets/30156cc0-bc12-49f7-a6d8-a1af191b6f5c)